nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -210,6 +222,7 @@ def input_types(self):
                 "cache_last_channel": NeuralType(('D', 'B', 'T', 'D'), ChannelType(), optional=True),
                 "cache_last_time": NeuralType(('D', 'B', 'D', 'T'), ChannelType(), optional=True),
                 "cache_last_channel_len": NeuralType(tuple('B'), LengthsType(), optional=True),
+                "pre_encode_input": NeuralType(tuple(), LogitsType(), optional=True),
```

Comment:
its not logits type. Maybe `EncodedRepresentation`?  Also add suggested shape

Reply:
Changed to `BoolType()`.